src/components/EventInfo/EventPage.js

```diff
@@ -0,0 +1,139 @@
+import React from 'react';
+import PropTypes from "prop-types";
+import { Link } from 'react-router-dom';
+import { connect } from 'react-redux';
+import * as moment from 'moment';
+import { emailToName, nameToInitial } from '../../utils/texts';
+
+// Material
+import Paper from '@material-ui/core/Paper';
+import { withStyles } from '@material-ui/core/styles';
+import Card from '@material-ui/core/Card';
+import CardActionArea from '@material-ui/core/CardActionArea';
+import CardActions from '@material-ui/core/CardActions';
+import CardContent from '@material-ui/core/CardContent';
+import CardMedia from '@material-ui/core/CardMedia';
+import Button from '@material-ui/core/Button';
+import Typography from '@material-ui/core/Typography';
+import Avatar from '@material-ui/core/Avatar';
+import Chip from '@material-ui/core/Chip';
+import theme from './../muiTheme';
+
+const styles = {
+    root: theme.root,
+    chipsContainer: {
+      ...theme.components.chipsContainer,
+      padding: theme.spacing.chipsSpacing,
+    },
+    chip: {
+      margin: theme.spacing.chipsSpacing,
+    },
+    card: {
+      width: theme.spacing.width
+    },
+    actions: {
+      width: theme.spacing.width
+    },
+    media: {
+      height: theme.spacing.mediaHeightSmall,
+      backgroundColor: theme.palette.primary1Color
+    },
+    container: {
+      ...theme.container
+    }
+};
+
+class EventPage extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render () {
+    const { events, calendar, classes } = this.props;
+    const selectedId = this.props.match.params.id;
+    const currentEvent = events.find(event => event.id == selectedId)
+    const eventStartTime = moment(currentEvent.start.dateTime).format('HH:MM');
+    const eventEndTime = moment(currentEvent.end.dateTime).format('HH:MM');
+    const personsWithCreator = [{ email: currentEvent.creator.email, responseStatus: 'accepted'}]
```

Comment:
I think you can also include here those whose acceptance is pending (a good point to discuss with people on meeting regarding that)